### PR TITLE
hexToBin/binToHex functionality already exists in openssl

### PR DESCRIPTION
**Pull request by DBarney**
the third parameter of the `openssl.digest.digest` function is a flag that can print out a binary string, or a hex string.

https://github.com/zhaozg/lua-openssl/blob/master/src/digest.c#L89

### CODE REVIEW

**Comment by creationix** [comment]
That's handy.

**Comment by creationix** [comment]
Is there nothing else using hex-bin?  I'm pretty sure it has several deps within lit.

**Comment by DBarney** [comment]
You were right, there were a few last places that it was mentioned. I removed them and replaced them with `require('openssl').hex`

File: deps/websocket-codec.lua [hunk 1]
```diff
@@ -1,8 +1,6 @@
 exports.name = "creationix/websocket-codec"
 exports.version = "1.0.5"
-exports.dependencies = {
-  "creationix/hex-bin@1.0.0"
-}
+exports.dependencies = {}
```
Comment (creationix) [inline deps/websocket-codec.lua]: No need for an empty table, you can just remove this entry.

**Pull request merged**